Resources/Prototypes/Recipes/Crafting/Graphs/improvised/soyplush.yml

```diff
@@ -0,0 +1,13 @@
+- type: constructionGraph
```

Comment:
I don't think it should be constructible, it kind of bloats the construction menu a bit too much and none of the other plushies work like that

Reply:
I'm in agreement. Keep it out of construction.